exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -143,13 +143,19 @@ func MakeDependencySubsegmentForLocalRootDependencySpan(span ptrace.Span, resour
 	// Make this a subsegment
 	dependencySubsegment.Type = awsxray.String("subsegment")
 
-	addNamespaceToSpansWithRemoteService(span, dependencySubsegment)
+	if dependencySubsegment.Namespace == nil {
+		dependencySubsegment.Namespace = awsxray.String("remote")
+	}
 
 	// Remove span links from consumer spans
 	if span.Kind() == ptrace.SpanKindConsumer {
 		dependencySubsegment.Links = nil
 	}
 
+	myAwsRemoteService, _ := span.Attributes().Get(awsRemoteService)
+
+	dependencySubsegment.Name = awsxray.String(myAwsRemoteService.Str())
```

Comment:
IMO I think it would be better to defensively check for nil:

```
// Set the name
if myAwsRemoteService, ok := span.Attributes().Get(awsRemoteService); ok {
   serviceSegment.Name = awsxray.String(myAwsRemoteService.Str())
}
```